lib/gui/app.js

```diff
@@ -85,23 +86,44 @@ app.run(() => {
 app.run((AnalyticsService, ErrorService, UpdateNotifierService, SelectionStateModel) => {
   AnalyticsService.logEvent('Application start');
 
-  if (UpdateNotifierService.shouldCheckForUpdates() && !process.env.ETCHER_DISABLE_UPDATES) {
-    AnalyticsService.logEvent('Checking for updates');
+  const shouldCheckForUpdates = UpdateNotifierService.shouldCheckForUpdates();
 
-    UpdateNotifierService.isLatestVersion().then((isLatestVersion) => {
+  if (!shouldCheckForUpdates || process.env.ETCHER_DISABLE_UPDATES) {
+    AnalyticsService.logEvent('Not checking for updates', {
+      shouldCheckForUpdates,
+      ETCHER_DISABLE_UPDATES: process.env.ETCHER_DISABLE_UPDATES
```

Comment:
Yeah, sounds good to me. Let's write "disableUpdatesEnvironmentVariable" though, I really hate abbreviations in code :P